cmd/gateway-main.go

```diff
@@ -47,6 +47,10 @@ EXAMPLES:
 
   2. Start minio gateway server bound to a specific ADDRESS:PORT.
       $ {{.HelpName}} --address 192.168.1.101:9000 azure
+
+  3. Gateway server connecting to a specific Azure Blob Storage endpoint.
+      $ {{.HelpName}} azure https://storage.minio.io
```

Comment:
Is this a valid endpoint entry? `storage.minio.io` ? - we can make example closer how an actual entry would look like.